bigint/src/bigint.rs

```diff
@@ -994,6 +1044,23 @@ impl BigInt {
     pub fn to_bytes_le(&self) -> (Sign, Vec<u8>) {
         (self.sign, self.data.to_bytes_le())
     }
+    
+    /// Returns the two's complement byte representation of the `BigInt` in little-endian byte order.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num_bigint::ToBigInt;
+    ///
+    /// let i = -1125.to_bigint().unwrap();
+    /// assert_eq!(i.to_twos_complement_bytes_le(), vec![155, 251]);
+    /// ```
+    #[inline]
+    pub fn to_twos_complement_bytes_le(&self) -> Vec<u8> {
+        let mut bytes = self.data.to_bytes_le();
+        twos_complement_le(&mut bytes);
```

Comment:
You should only flip it for `Sign::Minus`!

It will also need to consider overflow - e.g. -192 will give you biguint `[0xc0]`, complemented should be `[0x40,0xff]`.